web-marketplace/src/legacy-pages/CreateOrganization/CreateOrganization.tsx

```diff
@@ -67,12 +68,25 @@ function CreateOrganizationContent({
     handleActiveStep,
     activeStep,
     percentComplete,
-    handleSaveNext,
     isLastStep,
     data,
     handleResetData,
   } = useMultiStep<OrganizationMultiStepData>();
 
+  // Refs for each form step
+  const orgProfileFormRef: FormRef = useRef(undefined);
+  const migrateProjectsFormRef: FormRef = useRef(undefined);
+  const personalInfoFormRef: FormRef = useRef(undefined);
+  const inviteMembersFormRef: FormRef = useRef(undefined);
```

Comment:
there's not one specific form on the invite members step so I think we can skip this for invite members

Reply:
or we just keep it here, but he'll stay undefined, which could be a bit confusing